adapters/vox/vox.go

```diff
@@ -0,0 +1,63 @@
+package vox
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/prebid/openrtb/v19/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type voxAdapter struct {
```

Comment:
Please rename this struct to `adapter` instead of `voxAdapter` as the scope of this struct is limited to this package only.

Reply:
Done.